GitCommands/Settings/ConfigFileSettings.cs

```diff
@@ -70,15 +70,11 @@ public static ConfigFileSettings CreateGlobal(ConfigFileSettings? lowerPriority,
                 SettingLevel.SystemWide);
         }
 
-        public new string GetValue(string setting)
-        {
-            return GetString(setting, string.Empty);
-        }
+        public new string GetValue(string setting) => GetString(setting, string.Empty);
 
-        public IReadOnlyList<string> GetValues(string setting)
-        {
-            return SettingsCache.GetValues(setting);
-        }
+        public bool IsEnabled(string setting) => GetValue(setting).Trim().Equals("true", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
```suggestion
```
I.e. no need to touch this file, or else thoroughly.

Reply:
Agree, I'd like to avoid increases of the public API surface here.